### PR TITLE
Insert at scale

### PR DESCRIPTION
**Pull request by kapolyak**
Second try on correct master. Relevant files are loadDataPSQ.js, loadDataMDB.js, and generateListingsArray.js (for postgres only)

### CODE REVIEW

**Comment by benmangold** [comment]
on it

File: database/index.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ const hstore = require('pg-hstore');
 const db = new Sequelize('neighborhood', 'root', 'hrnyc18',
   {
     dialect: 'postgres',
+    pool: {
+      max: 10,
+      min: 1,
+      idle: 10000
+    },
+    logging: false
```
Comment (benmangold) [inline database/index.js:14]: any thoughts on logging? as a team we should probably explore and discuss this

File: server/models/dummyData/generateListingsArray.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const faker = require('faker');
+
+////////// POSTGRES ONLY //////////
+
+const generateDummyArray = () => {
+  return new Promise((resolve, reject) => {
+    var scaleListingsArray = [];
+    for (var j = 0; j < 10000; j++) {
```
Comment (benmangold) [inline server/models/dummyData/generateListingsArray.js:8]: 10000 could assigned to and used as const NUM_ARRAYS (or something) for readability

File: database/index.js (hunk 1 above)
Comment (kapolyak) [reply database/index.js:14]: this is to stop overly-comprehensive logging from postgres, but totally agree we should discuss logging as a team for proxies

File: server/models/dummyData/generateListingsArray.js (hunk 2 above)
Comment (kapolyak) [reply server/models/dummyData/generateListingsArray.js:8]: good note, thanks!

File: database/index.js (hunk 1 above)
Comment (benmangold) [reply database/index.js:14]: discussing logs, we all want to align what service we're using
maybe loggly, free 30 day trial
data dog, 14 days free, multiple servers

likely won't get to this until after the holidays

https://www.loggly.com/signup/

https://www.datadoghq.com/

File: server/models/loadDataMDB.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      })
+    })
+  }
+
+  async function initialize() {
+    console.log('****** Begin Data Injection ******')
+    var begin = Date.now();
+    for (var i = 0; i < 1000; i++) {
+      var listingArray = await generateDummyArray(i)
+      await insertRecs(listingArray);
```
Comment (benmangold) [inline server/models/loadDataMDB.js:66]: nice async work here, cool to see async/await at work with promises.  seems to align with what's expressed in https://medium.com/@bluepnume/even-with-async-await-you-probably-still-need-promises-9b259854c161

^ also lots of possible styles to draw from in structuring promises and async await together

File: server/models/loadDataPSQL.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+          console.log('Current total records:', totalAdded);
+          resolve();
+        })
+        .catch((error) => {
+          reject(error);
+        })
+    })
+  }
+
+  async function initialize() {
```
Comment (benmangold) [inline server/models/loadDataPSQL.js:31]: same thing, cool to see async await at work with promises

File: server/models/loadDataMDB.js (hunk 3 above)
Comment (benmangold) [reply server/models/loadDataMDB.js:66]: https://twitter.com/i/web/status/930108010558640128 this in particular is nuts

**Pull request merged**